packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-development.md

```diff
@@ -1,116 +1,125 @@
 ---
-id: Product development guide
+id: Develop accessibly
 section: accessibility
 ---
 import { AccessibilityDatalist } from './accessibility-datalist.js';
 
-## General questions to consider
+## Prioritizing accessibility during development
 
-To assess how accessible your applications or products are, consider these questions:
-- **Is it discoverable/perceivable by all users?** If you can see or click on it, then it should be discoverable and perceivable by all users. You should determine if you can easily navigate to it by keyboard or by screen reader.
-- **Is it interactable?** If you can get to it, it should be easy to actually use once you’ve focused on the element. You should be able to interact with it by keyboard (for example, if it’s a button, can you press **Enter** to initiate the action?). Also use a screen reader to ensure you can initiate the action.
-- **Is it understandable?** If the element can be found and interacted with, it should be clear what this action does. If it’s a button, you should determine if it has visible text that would be clear out of context of the page. If it doesn’t have visible text, you should check if it has an aria-label or accessible name.
+As you develop your product, it can be helpful to keep accessibility top of mind, by asking yourself a few questions: 
 
+**1. Can all users discover and perceive elements?**
 
-## What products should address
+If you can see or click on it, then it should be discoverable and perceivable by all users, including those using a keyboard or screen reader.
```

Comment:
Do you think it's clear that discoverable/perceivable means users can navigate to the element? When I wrote that second sentence, mostly I was trying to say it a different way to make sure that the audience understood what I meant. If you think it's clear, I'm totally good with this change, but just want to make sure. :)

Reply:
Maybe adding in a ", and thus navigable" or something?

Only additional thing I have to add here is "...including those using a keyboard or other assistive technology such as a screen reader."